common/components/display/template.njk

```diff
@@ -0,0 +1,16 @@
+{% from "items/macro.njk" import appItems %}
+
+<div class="app-display govuk-form-group {%- if params.classes %} {{ params.classes }}{% endif %}">
```

Comment:
What about `readOnlyField` as the component name? This component is specifically adding a field so it would be good to be clearer about what its doing. "Display" is a bit ambiguous and could be related to anything in the view, rather than specifically displaying a field as uneditable.

Reply:
Done